packages/design-system/src/components/tooltip.tsx

```diff
@@ -85,6 +99,102 @@ export const Tooltip = React.forwardRef(function TooltipWrapper(
   );
 });
 
+/**
+ * EnhancedTooltip has the following differences from the radix-ui
+ * 1. Don't show the tooltip if any click or key-down was made inside Tooltip.Trigger
+ * 2. Show Tooltip on focus after the delay
+ **/
+export const EnhancedTooltip = React.forwardRef(
+  function TextControlTooltipWrapper(
```

Comment:
doesn't really need a name

Reply:
eslint display name rule, changed on Enhanced...

Reply:
The inner function will not help react dev tools afaik, you still need displayName in the end and everywhere else linter was fine